impl/src/main/java/io/jsonwebtoken/impl/BackwardCompatibilityUtil.java

```diff
@@ -0,0 +1,26 @@
+package io.jsonwebtoken.impl;
+
+import io.jsonwebtoken.lang.Services;
+import io.jsonwebtoken.lang.UnavailableImplementationException;
+import io.jsonwebtoken.lang.UnknownClassException;
+
+/**
+ * A package private utility class to help retain backward compatibility, before breaking changes are made for v1.0.
+ * TODO: remove this class before v1.0
+ */
+@Deprecated
+final class BackwardCompatibilityUtil {
```

Comment:
This class name is ugly, but I don't have any better ideas.  Moved to a util class to make testing (coverage) easier

Reply:
How about `LegacyServices.loadFirst` ? And put it in the `io.jsonwebtoken.impl.lang` package to mirror the current `Services` class?  This way also ensures the method signature is the same, so when refactoring comes, we just remove the `Legacy` prefix on the respective line of code.

Reply:
Much better name! And I'm also less afraid to stick it in `.lang` with a name like that too